components/Template.js

```diff
@@ -69,4 +67,44 @@ function transformData(fn, templateKey, originalData) {
   throw new Error('`transformData` must be a function or an object');
 }
 
+function renderTemplate({template, compileOptions, helpers, data}) {
+  var hogan = require('hogan.js');
+  var forOwn = require('lodash/object/forOwn');
+  var content;
+
+  if (typeof template !== 'string' && typeof template !== 'function') {
```

Comment:
Even though `typeof` is pretty common, it's still not very readable. What about : 
```js
var isString = typeof template === 'string';
var isFunction = typeof template === 'function';

if(!isString && !isFunction) {
  throw new Error('Template must be `string` or `function`');
}
content = isFunction ? 
  template(data):
  hogan.compile(template, compileOptions).render(addTemplateHelpersToData(data));

```